app_remote/flaskr/static/js/app.js

```diff
@@ -0,0 +1,17 @@
+import { PiiAppClient } from './piiAppClient.js'
+
+const forwardTicketId = function (zafClient, appClient) {
+  zafClient.get('ticket.id')
+    .then(payload => appClient.replaceTicketId(payload))
+    .then(response => {
+      if (response.status !== 200) {
+        console.log(`Request failed. Status code: ${response.status}`)
```

Comment:
Having code execute itself upon import is not good practice as it makes it hard to test those functions. Better to import a function, and then execute it in the script tag

Not a great example, however see how GA does its importing https://developers.google.com/analytics/devguides/collection/analyticsjs.

It first imports a function, and then executes the function once imported to trigger it. 

This allows a few things
- Testability: You can easily write a test around a function, much harder to write tests around code that executes itself on import
- Flexibility of use: If this code should wait on some sort of event before triggering, it can be re-used more easily if wrapped in a function

Reply:
I think this is addressed in the earlier comment, but likely I may miss something so let me know.